src/plays/calendar/Calendar.scss

```diff
@@ -0,0 +1,342 @@
+$vincentBCP-calendar-box-shadow: 0px 0px 10px 5px rgba(0,0,0,0.1);
```

Comment:
I would recommend separating the color from the shadow config. so you would end up like
```javascript
$calendar-box-shadow-dark: 0px 0px 10px 5px rgba(0,0,0,0.1);
$vincentBCP-calendar-box-shadow: 0px 0px 10px 5px $calendar-box-shadow-dark;
```